pre_commit/git.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import os.path
 import sys
+from pathlib import Path
```

Comment:
I would prefer not to use pathlib

you can do the same without it via `os.path.commonpath`:

```python
root = os.path.realpath(root)
git_dir = os.path.realpath(git_dir)
if os.path.commonpath(root, git_dir) == git_dir:
    ...
```